tests/rules/test_gcloud_cli.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+
+from thefuck.rules.gcloud_cli import match, get_new_command
+from thefuck.types import Command
+
+no_suggestions = '''\
+ERROR: (gcloud) Command name argument expected.
+'''
+
+misspelled_command = '''\
+ERROR: (gcloud) Invalid choice: 'comute'.
+Usage: gcloud [optional flags] <group | command>
+  group may be           access-context-manager | ai-platform | alpha | app |
+                         asset | auth | beta | bigtable | builds | components |
+                         composer | compute | config | container | dataflow |
+                         dataproc | datastore | debug | deployment-manager |
+                         dns | domains | endpoints | filestore | firebase |
+                         functions | iam | iot | kms | logging | ml |
+                         ml-engine | organizations | projects | pubsub | redis |
+                         resource-manager | scheduler | services | source |
+                         spanner | sql | tasks | topic
+  command may be         docker | feedback | help | info | init | version
+
+For detailed information on this command and its flags, run:
+  gcloud --help
+'''
```

Comment:
`misspelled_command` is not used in any test case. Also, `match` does not consider that type of output. You can use other rules as a reference – such as `docker_not_command` – which uses string matching and splitting. Other rules – like `react_native_command_unrecognized` – use regular expression. I'd recommend going for string matching and splitting. What do you think?

Reply:
Yeap ill look into those examples and make changes